language-php.md

```diff
@@ -8,6 +8,69 @@ We decided to follow [PSR-2].
 [1]: http://www.phptherightway.com/
 [PSR-2]: https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md
 
+## SQL Queries
+MySQL keywords are always capitalized: SELECT, INSERT, UPDATE, WHERE, AS, JOIN, ON, IN, etc. Break up long queries into multiple lines for legibility, preferably breaking for each clause.
```

Comment:
Honestly I see no difference between _sql itself_ and _sql when used in PHP_. Consider this: a lot of JavaScript is written inline in HTML. When writing a guideline for JavaScript, do you write it in the HTML document or the JavaScript document? It should be fairly clear that you write it in the latter. Point is, conventions are written to cover a _broad_ area, it just doesn't make sense to write a _narrow_ one: it would often be redundant. If I agree with you that, yeah, this is how we should write SQL but only when using PHP, it would be a repetitive task to write another document for SQL in Python, or any other language we might use in the future.

Plus, we should probably be using the ORM shipped with the framework and avoid writing SQL directly.